packages/react-magma-dom/src/components/Tabs/TabsScrollSpyContainer.test.js

```diff
@@ -0,0 +1,190 @@
+    const tab3 = container.querySelector('button[data-scrollspy-id="third"]');
+
+    fireEvent.click(tab1);
+
+    expect(tab1).toHaveStyleRule('color', magma.colors.primary500);
+    expect(tab2).toHaveStyleRule('color', magma.colors.neutral500);
+    expect(tab3).toHaveStyleRule('color', magma.colors.neutral500);
+
+    fireEvent.click(tab2);
+
+    expect(tab1).toHaveStyleRule('color', magma.colors.neutral500);
+    expect(tab2).toHaveStyleRule('color', magma.colors.primary500);
+    expect(tab3).toHaveStyleRule('color', magma.colors.neutral500);
+
+    fireEvent.click(tab3);
+
+    expect(tab1).toHaveStyleRule('color', magma.colors.neutral500);
+    expect(tab2).toHaveStyleRule('color', magma.colors.neutral500);
+    expect(tab3).toHaveStyleRule('color', magma.colors.primary500);
+  });
+
+  it('Should set the tabLabel', () => {
+    const { getByText } = render(
+      <TabsScrollSpyContainer activeIndex="" testId={testId}>
+        <TabScrollSpyPanel tabLabel={'first'}>
+          Content Area One
+        </TabScrollSpyPanel>
+      </TabsScrollSpyContainer>
+    );
+
+    expect(getByText('first')).toBeVisible();
+  });
+
+  it('Should set the icon', () => {
+    const { container } = render(
+      <TabsScrollSpyContainer activeIndex="" testId={testId}>
+        <TabScrollSpyPanel tabLabel={'first'} icon={<AndroidIcon />}>
+          Content Area One
+        </TabScrollSpyPanel>
+      </TabsScrollSpyContainer>
+    );
+
+    const tab1 = container.querySelector('button[data-scrollspy-id="first"]');
+
+    expect(tab1.firstChild.firstChild.nodeName).toBe('svg');
+  });
+
+  describe('Click events', () => {
+    it('Should allow a custom onClick function', () => {
+      const customOnClick = jest.fn();
+
+      const { getByTestId } = render(
+        <TabsScrollSpyContainer
+          testId={`${testId}-Tab`}
+          onClick={customOnClick}
+        ></TabsScrollSpyContainer>
+      );
+
+      fireEvent.click(getByTestId(`${testId}-Tab`));
+      expect(customOnClick).toHaveBeenCalledTimes(1);
+    });
+
+    it('Should set the window.location.href on click', () => {
+      const { container } = render(
+        <TabsScrollSpyContainer testId={testId}>
+          <TabScrollSpyPanel tabLabel={'first'}>
+            Content Area One
+          </TabScrollSpyPanel>
+        </TabsScrollSpyContainer>
+      );
+
+      const tab3 = container.querySelector('button[data-scrollspy-id="first"]');
+
+      fireEvent.click(tab3);
+
+      expect(window.location.href).toInclude('#first');
+    });
+
+    it('Should prevent disabled from being clickable', () => {
+      const { container } = render(
+        <TabsScrollSpyContainer testId={testId}>
+          <TabScrollSpyPanel tabLabel={'first'} disabled>
+            Content Area One
+          </TabScrollSpyPanel>
+        </TabsScrollSpyContainer>
+      );
+      const tab1 = container.querySelector('button[data-scrollspy-id="first"]');
+
+      expect(tab1).toHaveAttribute('disabled');
```

Comment:
This proves that it's disabled, but doesn't prove that it can't be clicked

Reply:
`disabled` removed entirely